deploy/devel/auth-patch.yaml

```diff
@@ -4,9 +4,11 @@ metadata:
     name: basic-user-pass
     namespace: image-build
     annotations:
-        # this is the minikube host ip by default, you might have to change it
-        # (run `minikube ip` to see it on your system)
-        tekton.dev/docker-0: http://192.168.49.1
+        # CHANGEME
+        # this is the ip to access your host from minikube (run `minikube ssh`,
```

Comment:
This is not true, it's the IP to access harbor from minikube, it just happens that on some systems minikube is able to simplify that by offering the 'host.minikube.internal' alias + firewall rules.

For example, this this not work on my setup as minikube does not handle the firewall rules, so I have to use my laptop external ip instead.

Reply:
I agree that something like `IP to access harbor from minikube` is more accurate (and shorter 😛). I will reword this and add a note in the README on how to obtain this IP.

Reply:
I have refactored this removing the CHANGEME sections, and adding a more detailed explanation in the README.